docs/config-spec.md

```diff
@@ -17,73 +17,68 @@ For example:
 # Metadata
 #####################
 
-# TBD
-# Unique identifier for this observability pack
-# uuid: guid, required
+# A unique Observability Pack identifier
+uuid: guid, required
```

Comment:
This is not currently part of the json schema.

Reply:
What is this then? 


<img width="373" alt="Screen Shot 2021-08-19 at 8 47 44 AM" src="https://user-images.githubusercontent.com/1395158/130090006-00176514-2118-41d8-b278-e2fa5d79291c.png">